app/client/user/user.service.js

```diff
@@ -21,11 +21,15 @@ angular.module('app.user')
       return _this.singleRecordKey;
     };
 
-    this.getPage = function (skip, limit) {
+    this.getPage = function (skip, limit, sortBy, sortDirection) {
+
+      if (sortBy === undefined || sortBy === null) { sortBy = 'id'; }
+      if (sortDirection === undefined || sortDirection === null) { sortDirection = 'asc'; }
```

Comment:
this will be  a better line `if (!sortBy) {sortBy = 'id';}` and `if (!sortDirection) { sortDirection = 'asc'; }`